src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -920,33 +920,6 @@ public void updatedRequestHasUpdatedMetadata()
       metadata.getString("updatedDate"), is(not(createdMetadata.getString("updatedDate"))));
   }
 
-  @Test
```

Comment:
Has this behaviour changed?

Reply:
Yes.
The user id is no longer a required field in the metadata because trying to login should work without a user id.
https://github.com/folio-org/raml/pull/120/files
https://github.com/folio-org/raml-module-builder/pull/414/files

Therefore we no longer get a 500 but a 204 response code for a put where user id is null.

If we don't trust Okapi to only invoke modules when the headers are correct we may add some header validation to RMB RestVerticle, but this is a separate issue: https://issues.folio.org/browse/RMB-423
Okapi is in charge of checking whether calling a module without a valid user id 
https://github.com/folio-org/okapi/pull/294/files

Reply:
Thanks for explaining the thought process.

I agree that trusting clients and treating input defensively are separate and bigger topics, for another day.